shared/src/commonMain/kotlin/io/customer/shared/tracking/api/model/TrackingRequest.kt

```diff
@@ -0,0 +1,23 @@
+@file:UseContextualSerialization(Any::class)
+
+package io.customer.shared.tracking.api.model
+
+import io.customer.shared.common.CustomAttributes
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.UseContextualSerialization
+
+/**
+ * Request class to be used for individual tracking requests.
+ */
+// TODO: Find ways to break down into smaller classes and overcome kotlin type issue in serialization
+// of sealed classes with generic parsing
+@kotlinx.serialization.Serializable
+internal class TrackingRequest(
+    @SerialName("type") val type: String,
+    @SerialName("timestamp") val timestamp: Long? = null,
+    @SerialName("identifiers") val identifiers: CustomAttributes? = null,
+    @SerialName("attributes") val attributes: CustomAttributes? = null,
+    @SerialName("name") val name: String? = null,
+    @SerialName("device") val device: Device? = null,
+    @SerialName("metric") val metric: Metric? = null,
```

Comment:
```suggestion
    val type: String,
    val timestamp: Long? = null,
    val identifiers: CustomAttributes? = null,
    val attributes: CustomAttributes? = null,
    val name: String? = null,
    val device: Device? = null,
    val metric: Metric? = null,
```

I don;t see need for annotation

Reply:
Responded above, won't be addressing it in this PR.